### DISCUSSION TITLE
Does netdata now support service maps?

### DISCUSSION THREAD

**Discussion by bernersyang** [netdata/netdata#14901] [Q&A]
I saw an issue in 2018 related to the network topology. 
https://github.com/netdata/netdata/issues/5062

**Comment by akshayamadhuri** [comment]
According to the Netdata documentation, service maps are supported as of version 1.26.0, which was released in June 2021. Service maps provide a visual representation of the relationships between different services running on a system, allowing users to easily identify dependencies and troubleshoot issues.

To use service maps in Netdata, you will need to first enable the Netdata Agent API by setting enable_api_server = yes in the netdata.conf configuration file. Then, you can access the service map by navigating to the http://localhost:19999/api/v1/service-map endpoint in your web browser.

It's worth noting that service maps are still considered an experimental feature in Netdata, so they may not be fully stable or reliable in all environments.
